.template/package.json

```diff
@@ -4,12 +4,24 @@
   "description": "My first godspeed project.",
   "main": "dist/index.js",
   "scripts": {
-    "dev": "nodemon --watch src -e ts,js,yaml,json -r dotenv ./src/index.ts",
     "gen-crud-api": "npx @godspeedsystems/api-generator",
-    "build": "swc src -d dist && rsync -a --exclude '*.ts' ./src/ ./dist/ && rsync -a ./config/ ./dist/config && rsync -a ./src/mappings/ ./dist/mappings && rsync -a ./src/plugins/ ./dist/plugins",
-    "preview": "node dist/index.js",
-    "serve": "nodemon --watch src --watch config --watch mappings --watch plugins -e ts,js,yaml,json,.env --exec 'npm run build && npm run preview'",
-    "clean": "rm -rf dist"
+    "dev": "run-script-os",
+    "dev:win32": "nodemon --watch src -e ts,js,yaml,json -r dotenv src\\index.ts",
+    "dev:darwin:linux": "nodemon --watch src -e ts,js,yaml,json -r dotenv ./src/index.ts",
+    "dev:default": "nodemon --watch src -e ts,js,yaml,json -r dotenv ./src/index.ts",
+    "build":"run-script-os",
+    "build:win32":"swc src -d dist & rsync -a --exclude '*.ts' .\\src\\ .\\dist\\ & rsync -a .\\config\\ .\\dist\\config & rsync -a .\\src\\mappings\\ .\\dist\\mappings & rsync -a .\\src\\plugins\\ .\\dist\\plugins",
```

Comment:
windows doesn't have rsync command. it gives error.

Reply:
Done replaced with copyfiles package which supports windows os, gotta test with windows machine.